system-tests/lib/cre/capabilities/evm/evm.go

```diff
@@ -0,0 +1,41 @@
+package evm
+
+import (
+	"fmt"
+	chainselectors "github.com/smartcontractkit/chain-selectors"
+	"github.com/smartcontractkit/chainlink/system-tests/lib/cre"
+	"github.com/smartcontractkit/chainlink/system-tests/lib/cre/flags"
+	"strconv"
+
+	capabilitiespb "github.com/smartcontractkit/chainlink-common/pkg/capabilities/pb"
+
+	kcr "github.com/smartcontractkit/chainlink-evm/gethwrappers/keystone/generated/capabilities_registry_1_1_0"
+	keystone_changeset "github.com/smartcontractkit/chainlink/deployment/keystone/changeset"
+)
+
+var EVMCapabilityFactory = func(chainID uint64, chainFamily string) func(donFlags []string) []keystone_changeset.DONCapabilityWithConfig {
+	return func(donFlags []string) []keystone_changeset.DONCapabilityWithConfig {
+		var capabilities []keystone_changeset.DONCapabilityWithConfig
+
+		if flags.HasFlag(donFlags, cre.EVMCapability) {
+			selector, err := chainselectors.SelectorFromChainId(chainID)
+			if err != nil {
+				fmt.Printf("Error getting selector from chainID: %d, err: %s\n", chainID, err.Error())
+				selector = 0
+			}
+			labelledName := "evm" + ":ChainSelector:" + strconv.FormatUint(selector, 10)
+			fmt.Println("EVM Capability Labelled Name:", labelledName)
+			capabilities = append(capabilities, keystone_changeset.DONCapabilityWithConfig{
+				Capability: kcr.CapabilitiesRegistryCapability{
+					LabelledName:   labelledName,
+					Version:        "1.0.0",
+					CapabilityType: 3, // TARGET
```

Comment:
I need to add this capability 1 more time, but with `CapabilityType: 0, // TRIGGER` (like `cron`), but having the same ID will probably collide

Reply:
did you solve that issue somehow?

Reply:
seems that the log trigger registration worked, but I'm still figuring out (right now I'm more keen in having the actions work and then keep evolving the solution)